client/packages/invoices/src/Prescriptions/DetailView/Payments/PaymentsModal.tsx

```diff
@@ -61,23 +67,69 @@ export const PaymentsModal: FC<PaymentsModalProps> = ({
           }}
         />
       }
+      sx={{
+        '& .MuiDialogContent-root': { display: 'flex', alignItems: 'center' },
+      }}
     >
-      <Grid container spacing={3} justifyContent="center">
-        {fields.map(({ label, value, disabled = false, onChange }, index) => (
-          <Grid key={index} size={4}>
-            <InputWithLabelRow
-              label={label}
-              Input={
-                <CurrencyInput
-                  value={value}
-                  disabled={disabled}
-                  onChangeNumber={onChange}
-                />
+      <Stack gap={2}>
```

Comment:
Can you change this to be more like the screen shot shared in this issue?
https://github.com/msupply-foundation/open-msupply/issues/6524#issuecomment-2664511778

I think we need to go back to a 2 column grid, rather than Stack...